### PR TITLE
added unit test for Document type Will

### PR DESCRIPTION
**Pull request by vakepati**
### JIRA link (if applicable) #397 
https://tools.hmcts.net/jira/browse/BPS-448
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/MetafileJsonValidatorTest.java [hunk 1]
```diff
@@ -24,8 +24,8 @@ public void setUp() throws IOException, ProcessingException {
     }
 
     @Test
-    public void should_successfully_map_json_file_to_entities() throws IOException {
-        InputEnvelope envelope = getEnvelope("/metafiles/valid/from-spec.json");
+    public void should_successfully_map_sscs_json_file_to_entities() throws IOException {
```
[the remark this answers is not in the document]
Comment (vakepati) [reply src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/MetafileJsonValidatorTest.java]: As we discussed currently Jurisdictions will allow any String values as per the json schema https://github.com/hmcts/bulk-scan-processor/pull/402/files#diff-8718071a5731b2035c4f4541d9b2a95fL19,It is not restricting to any specific values for Jurisdictions.So even if we write a single Test to cover Jurisdictions which will not verify all possible scenarios.

File: src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/MetafileJsonValidatorTest.java [hunk 2]
```diff
@@ -42,6 +42,26 @@ public void should_successfully_map_json_file_to_entities() throws IOException {
...
+        assertThat(envelope.payments).hasSize(1);
+        assertThat(envelope.payments.get(0).amount).isEqualTo(new BigDecimal("100.00"));
+        assertThat(envelope.caseNumber).isEqualTo("1111222233334459");
+        assertThat(envelope.classification).isEqualTo(Classification.SUPPLEMENTARY_EVIDENCE);
+        assertThat(envelope.scannableItems)
+            .extracting("documentType")
+            .containsExactlyInAnyOrder(
+                InputDocumentType.CHERISHED,
+                InputDocumentType.OTHER,
+                InputDocumentType.WILL
```
Comment (lgonczar) [inline src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/MetafileJsonValidatorTest.java]: This document type should be added to the previous test and this test as such should be removed because it repeats a lot of assertions that are not related with jurisdiction.

File: src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/MetafileJsonValidatorTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (lgonczar) [reply src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/MetafileJsonValidatorTest.java]: We can still write that test to make sure all known jurisdictions are accepted. And when we improve validation, we will also add a test that checks if other jurisdictions are rejected.
